nautobot_circuit_maintenance/handle_notifications/handler.py

```diff
@@ -164,6 +164,18 @@ def create_or_update_circuit_maintenance(
     maintenance_id = f"{raw_entry.provider.slug}-{parser_maintenance.maintenance_id}"
     try:
         circuit_maintenance_entry = CircuitMaintenance.objects.get(name=maintenance_id)
+        last_parsed_notification = (
+            circuit_maintenance_entry.parsednotification_set.order_by("raw_notification__date").reverse().last()
```

Comment:
I see that both `ParsedNotification` and `RawNotification` have a `date` field - for my education, what's the difference between these two fields' expected values? Does sorting the `ParsedNotification` based on the associated `raw_notification__date` result in a different expected ordering than sorting by its own `date` field?

Reply:
The ParsedNotification.date is the date when the notification was parsed, the RawNotification.date is the date when the notification was sent.

if it's too confusing we may change the meaning...

Reply:
That is a bit confusing - it would be good at least to clearly document the meaning of these fields in `models.py` and wherever they're being referenced, like here.